.gitignore

```diff
@@ -0,0 +1,4 @@
+/implementation/target/
```

Comment:
Should only need "target" as opposed to specific directories.

Take a look at the other projects for examples

Reply:
I copied the gitignore from Health to make sure we have all the known ignores.